docs/src/main/asciidoc/kotlin.adoc

```diff
@@ -20,7 +20,7 @@ To complete this guide, you need:
 * JDK 1.8+ installed with `JAVA_HOME` configured appropriately
 * Apache Maven 3.5.3+
 
-
+NB: For Gradle project setup please see below
```

Comment:
Could we add a link to the referenced `grandle` section?

Reply:
Added a link to the Gradle setup page.